_source/_posts/2021-06-07-session-mgmt-node.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: blog_post
+title: "Everything You Ever Wanted to Know About Session Management in Node.js"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "A quick run-down of how to manage sessions in Node.js."
+tags: [session, node]
+tweets:
+- "Need to learn how to manage sessions in Node? We've got you covered!"
+- "Learn how to manage sessions in Node!"
+- "Managing sessions in Node can be confusing. Check this out!"
+image: blog/featured/okta-node-skew.jpg
+type: conversion
```

Comment:
This doesn't seem like conversion since there's no reference to http://developer.okta.com/signup nor a cli.md include.